patches/PRODSECBUG-2198__preauth_sql__2.2.0.patch

```diff
@@ -0,0 +1,96 @@
+diff --git a/vendor/magento/module-catalog/Model/Product/ProductFrontendAction/Synchronizer.php b/vendor/magento/module-catalog/Model/Product/ProductFrontendAction/Synchronizer.php
```

Comment:
Change first line to be same as our other patches with parameters. Also drop index